File: data/python/MAXIMUM_AREA_RECTANGLE_PICKING_FOUR_SIDES_ARRAY_TRANSLATED.java
====================
public static int f_gold ( int [ ] arr , int n ) {
  Arrays . sort ( arr , 0 , n ) ;
  int [ ] dimension = {
    0 , 0 };
    int i = 0 ;
    int j = 0 ;
    while ( ( i < n - 1 && j < 2 ) || ( i < n - 1 && j < 2 ) ) {
      if ( ( arr [ i ] == arr [ i + 1 ] ) && ( arr [ j ] == arr [ i ] ) ) {
        dimension [ j ] = arr [ i ] ;
        j ++ ;
        i ++ ;
      }
      i ++ ;
    }
    return ( dimension [ 0 ] * dimension [ 1 ] ) ;
  }
  

File: data/java/MAXIMUM_AREA_RECTANGLE_PICKING_FOUR_SIDES_ARRAY_TRANSLATED.py
====================
def f_gold ( arr , n ) :
    arr = np.asarray ( arr )
    dimension = [ 0 , 0 ]
    for i , j in enumerate ( range ( n - 1 , 2 ) ) :
        if arr [ i ] == arr [ i + 1 ] :
            dimension [ j ] = arr [ i ]
    return ( dimension [ 0 ] * dimension [ 1 ] , dimension [ 1 ] * dimension [ 2 ] )

